.bazelrc

```diff
@@ -1,3 +1,7 @@
+# Required for bazel 3.4+
+# See https://github.com/bazelbuild/bazel/issues/10674#issuecomment-658270895
+build --incompatible_objc_compile_info_migration
```

Comment:
damn this migration thing keep coming up :(

Reply:
It's a good thing though! the objc provider is bad, and consolidation with CcInfo will make things simpler in the long run

Reply:
totally agree!